### ISSUE TITLE
[modificar costo] Wamos

### ISSUE THREAD

**Issue by GuadaVivas** [toselli/issues#709]
### Descripcion del error

HOla chicos,
Necesitamos modificar el PVP de a siguiente reserva a **3180 usd**
Gracias!

### Paso a paso para reproducir el error

.

### Captura del error

.

### Link al error y/o Localizador

https://adm.ebooking.com.ar/bookings/detail/639dc3f8332498c51593fe50?purchaseGroupId=453448#heading_639dc3f8332498c51593fe50

**Comment by gabrielmarinsaldaJT** [comment]
Hola chicos

El costo es igual? 

Saludos

**Comment by mvalentinalucero** [comment]
Hola Gabi, a que haces referencia? El precio sugerido de venta al publico de acuerdo lo que indica la vendedora deberia ser de 3180 usd

**Comment by gabrielmarinsaldaJT** [comment]
Claro, yo puedo modificar el vosto de PVP pero la rentabilidad va a dar mal

Solamente tienen que modificar eso?

**Comment by mvalentinalucero** [comment]
Hola Gabi! Yo le modifique el costo a pagar al proveedor, y ahora solicita modificar el pvp asique entiendo seria solamente eso
